docker-gammasim-tools-dev/external/README.md

```diff
@@ -0,0 +1,28 @@
+# External resources accessible by the Container
+
+Install here any software / modules requires inside the container.
+
+gammasim-tools
+```
+git clone https://github.com/gammasim/gammasim-tools.git
+```
+
+For running without the model DB:
+
+1. model configuration
+```
+git clone https://gitlab.cta-observatory.org/cta-consortium/aswg/simulations/simulation-model/simulation-model-description.git
+``` 
+2. verification process files
+```
+mkdir -p verification-process
+cd verification-process
+git clone https://gitlab.cta-observatory.org/cta-consortium/aswg/simulations/simulation-model/verification/verification-process/mst-structure.git
+git clone https://gitlab.cta-observatory.org/cta-consortium/aswg/simulations/simulation-model/verification/verification-process/sst-structure.git
+git clone https://gitlab.cta-observatory.org/cta-consortium/aswg/simulations/simulation-model/verification/verification-process/mst-flashcam.git
+git clone https://gitlab.cta-observatory.org/cta-consortium/aswg/simulations/simulation-model/verification/verification-process/lst.git
+```
+3. instrument files (not yet on gitlab?)
+```
+from svn: dataFromInstrumentTeams
```

Comment:
All instrument files are on gitlab, but it's a nightmare to download, so we can probably leave it like this...

Reply:
We need to solve this type of gitlab issues, but this is a general CTA / gitlab question.